browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -193,12 +193,6 @@ class AdblockCnameResolveHostClient : public network::mojom::ResolveHostClient {
 void OnBeforeURLRequestAdBlockTP(const ResponseCallback& next_callback,
                                  std::shared_ptr<BraveRequestInfo> ctx) {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
-  // If the following info isn't available, then proper content settings can't
-  // be looked up, so do nothing.
-  if (ctx->tab_origin.is_empty() || !ctx->tab_origin.has_host() ||
```

Comment:
mb we can DCHECK that `initiator_url` and `request_url`? To state explicit preconditions for the check.
And comment that tab_origin could be empty for SW